mkdocs.yml

```diff
@@ -0,0 +1,86 @@
+site_name: "MCAP"
+theme:
+  name: material
+  favicon: img/favicon32.png
+  logo: img/mcap.png
+  palette:
+    # Palette toggle for dark mode
+    - scheme: slate
```

Comment:
Can we make it light mode by default?

Reply:
Time to switch Studio over as well? 😁

Reply:
That is a debate as old as time 😄

In our community poll people seemed to favor dark mode. Let's at least start by keeping all our marketing sites light mode though.